src/onnx/parse_loop.cpp

```diff
@@ -54,7 +54,7 @@ struct parse_loop : op_parser<parse_loop>
             auto arg_iters = args.at(0)->eval();
             if(not arg_iters.empty())
             {
-                max_iterations = arg_iters.at<int64_t>();
+                max_iterations = std::min(arg_iters.at<int64_t>(), max_iterations);
```

Comment:
I didn't completely understand this issue. For the SSD model, isn't there a loop termination condition, loop trip count won't matter in that case if it is very high?

Reply:
> I didn't completely understand this issue. For the SSD model, isn't there a loop termination condition, loop trip count won't matter in that case if it is very high? 

Since we don't use dynamic shapes for the loop it will become a tensor of the max size.